src/librbd/cache/ParentCacheObjectDispatch.cc

```diff
@@ -78,7 +78,9 @@ bool ParentCacheObjectDispatch<I>::read(
     return false;
   }
 
-  auto [object_off, object_len] = extents.front();
```

Comment:
Nit: could use just use lambda capture renaming below?

Reply:
@dillaman 
No sure if that would fix the scoping issue complained about it.
So that is why I took apart the scope created by the `[]`s.
I tried renaming before, but it does not really make this already complex block more legible.